common/utils/dates.ts

```diff
@@ -136,3 +141,8 @@ export function getDatesBetween({
   }
   return dateArray;
 }
+
+export function countDaysBetween(a: Date, b: Date): number {
+  const millisecondsInDay = 1000 * 60 * 60 * 24;
+  return Math.floor((a.valueOf() - b.valueOf()) / millisecondsInDay);
+}
```

Comment:
This is based on Moment, and replaces `a.diff(b, 'day')` – and the Moment implementation is pretty simple: https://github.com/moment/moment/blob/e96809208c9d1b1bbe22d605e76985770024de42/src/lib/moment/diff.js#L41-L43